src/tests/suite/daos_epoch_io.c

```diff
@@ -342,7 +342,6 @@ daos_test_cb_add(test_arg_t *arg, struct test_op_record *op,
 		 char **rbuf, daos_size_t *rbuf_size)
 {
 	print_message("add rank %u\n", op->ae_arg.ua_rank);
-	test_rebuild_wait(&arg, 1);
```

Comment:
Why not move the rebuild_wait() after reint_server()? Do we need to wait rebuild for reintegration case?

Reply:
probably fine to keep the rebuild wait here.
I just want to add the rebuild wait after exclude, because in daos_run_io_conf test there are lots of exlude+reint, that may cause co-located layout, so if we exclude one rank here that may cause multi-shards failure, that may cause degraded fetch failure.

Reply:
i updated it a little bit.